libcxx/utils/generate_libcxx_cppm_in.py

```diff
@@ -9,11 +9,19 @@
 import os.path
 import sys
 
-from libcxx.header_information import module_c_headers, module_headers, header_restrictions, headers_not_available, libcxx_root
+from libcxx.header_information import (
```

Comment:
Reformatting looks a bit weird to me and I wonder whether we can avoid changing this part.
Does this script or format checking in CI fail to work without the trailing comma?